src/chi-vue/src/components/search-input/SearchInput.tsx

```diff
@@ -92,16 +93,24 @@ export default class SearchInput extends Vue {
         onClick={() => this.$emit(SEARCH_INPUT_EVENTS.SEARCH, this.inputValue)}
         aria-label="Search">
         <div class={BUTTON_CLASSES.CONTENT}>
-          <i class={`${ICON_CLASS} icon-search`}></i>
+          <i
+            class={`
+          ${ICON_CLASS} icon-search
+          ${this.portal && '-icon--primary'}
+          `}></i>
         </div>
       </button>
     );
 
     return (
-      <div class={`${INPUT_CLASSES.WRAPPER} ${INPUT_CLASSES.ICON.RIGHT}`}>
-        {searchInputElement}
-        {searchXIcon}
-        {searchIcon}
+      <div class="chi-toolbar__search">
+        <div class="chi-form__item">
+          <div class={`${INPUT_CLASSES.WRAPPER} ${INPUT_CLASSES.ICON.RIGHT}`}>
+            {searchInputElement}
+            {searchXIcon}
+            {searchIcon}
+          </div>
+        </div>
```

Comment:
```suggestion
const formItem = (
  <div class="chi-form__item">
    <div class={`${INPUT_CLASSES.WRAPPER} ${INPUT_CLASSES.ICON.RIGHT}`}>
      {searchInputElement}
      {searchXIcon}
      {searchIcon}
    </div>
  </div>;
);
const toolbar = (
  <div class="chi-toolbar__search">
    {formItem}
  </div>
);
return this.$props.dataTableSearch ? toolbar : formItem;
```

Please, keep in mind that the Search Input is supposed to work in Data Table Toolbar but also as a standalone independent component

Reply:
Also, please use the constants from classes.ts for these classes